services/DatasetsService.js

```diff
@@ -19,6 +19,7 @@ export default class DatasetsService {
       application: applications.join(','),
       language: this.opts.language,
       ...!!includes && { includes },
+      'user.role': 'ADMIN',
```

Comment:
For now, it is used only in the Backoffice but we should pass this as param from the component.

Reply:
👍